lib/routes/sms.js

```diff
@@ -106,8 +106,19 @@ module.exports = (log, isA, error, config, customs, sms) => {
           .then(reply, reply)
 
         function getLocation () {
+          if (request.app.clientAddress === '127.0.0.1') {
```

Comment:
Yeah, I know what you mean. I'm happy to do something different if you do come up with an idea...